#!/usr/bin/env python2

PACKAGE = "Vision"
# The following line should not be here for Groovy and above
import roslib;roslib.load_manifest(PACKAGE)

# For Groovy and above, the following line should be
# from dynamic_reconfigure.parameter_generator_catkin import *
from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("contourMinArea", int_t, 0, "Min contour area (1/1000)", 15, 0, 1000)
gen.add("hueLow",  int_t, 0, "Hue Low",   90, 0, 180)
gen.add("hueHigh", int_t, 0, "Hue High", 116, 0, 180)
gen.add("satLow",  int_t, 0, "Sat Low",    0, 0, 255)
gen.add("satHigh", int_t, 0, "Sat High", 255, 0, 255)
gen.add("valLow",  int_t, 0, "Val Low",    0, 0, 255)
gen.add("valHigh", int_t, 0, "Val High", 255, 0, 255)

exit(gen.generate(PACKAGE, "LaneMarkerDetector", "LaneMarkerDetector"))
